addon/components/frost-action-bar.js

```diff
@@ -164,8 +164,7 @@ export default Component.extend({
   // == Functions =============================================================
 
   controlIsVisible (control) {
-    const isVisible = this.convertControl(control).isVisible
-    return isVisible !== false || isVisible !== null
+    return this.convertControl(control).isVisible !== false
```

Comment:
Will the result of `this.convertControl(control).isVisible` be undefined? If a control doesn't have `isVisible` set, it seems like it might be missing that key. Could this be simplified as `return this.convertControl(control).isVisible`, since `undefined` is still falsey?

(It might not be legal to have a control declared that doesn't already have an `isVisible` property, though, in which case my concern is moot.)

Reply:
We want to ignore null/undefined values - essentially making true the default. Ideally the component would have a default value, but we don't know which component we may be handed, and the most common, `frost-button`, does not have a default value.